docs/_src/tutorials/tutorials/6.md

```diff
@@ -127,6 +127,43 @@ launch_milvus()
 document_store = MilvusDocumentStore()
 ```
 
+#### OPTION3: Elasticsearch
```

Comment:
Let's capitalize this in the same way as options 1 and 2

Reply:
done